include/net/net_pkt.h

```diff
@@ -240,6 +240,12 @@ struct net_pkt {
 	uint8_t ieee802154_arb : 1; /* ACK Request Bit is set in the frame */
 	uint8_t ieee802154_ack_fpb : 1; /* Frame Pending Bit was set in the ACK */
 	uint8_t ieee802154_frame_retry : 1; /* The frame is being retransmitted. */
```

Comment:
I thought the new parameters are to replace this one. Is there a need to keep `ieee_frame_retry` with more detailed data about frame counter, IEs, and if the frame is secured?

Reply:
From my point of view this flag can be removed.